test/NuGet.Tests.Apex/NuGet.Tests.Apex/NuGetIVsApiTests/NuGetConsoleTestCase.cs

```diff
@@ -4,6 +4,7 @@
 using Microsoft.Test.Apex.VisualStudio.Solution;
 using NuGet.Test.Utility;
 using Xunit;
+using System.Diagnostics;
```

Comment:
Nit: sort order, set VS to order system namespaces first and then sort/order them. Grab me if you need help with this.